sw/host/opentitantool/src/command/lc.rs

```diff
@@ -126,14 +126,17 @@ impl CommandDispatch for RawUnlock {
         // the LC TAP after the transition without risking the chip resetting.
         trigger_lc_transition(
             transport,
-            &mut *jtag,
+            jtag,
             DifLcCtrlState::TestUnlocked0,
             Some(token_words),
             /*use_external_clk=*/ true,
             self.reset_delay,
             /*reconnect_jtag_tap=*/ Some(JtagTap::LcTap),
         )?;
 
+        jtag = self.jtag_params.create(transport)?;
+        jtag.connect(JtagTap::LcTap)?;
```

Comment:
`trigger_lc_transition` should already reconnect to the LC TAP here, so won't this cause an issue?

Reply:
I changed `trigger_lc_transition` to consume the ownership of JTAG without reconnecting. I probably should rename `reconnect_jtag_tap` to `reset_jtag_strap` for clarity.